### ISSUE TITLE
Classify face/background and ROI tasks trained one after another

### ISSUE THREAD

**Issue by inspirit** [CongWeilin/mtcnn-caffe#4]
I remember you explained it partly why did you choose such train procedure:
1. train only classify task
2. fine-tune model with one step classify task followed by one step ROI task

what is the reason for such training?
in original paper they train model with all tasks together but with different weight for loss layers:
so they have classify+ROI weight = 1 and Landmarks weight = 0.5

thank you.

**Comment by CongWeilin** [comment]
It can make convergence easier with a initial weigh. prevent converge before learning the true feature

**Comment by inspirit** [comment]
can you please provide step by step train/tune for 48net? 
this one seems to be more complicated compared to 12/2nets.

**Comment by CongWeilin** [comment]
similar to 12net，add landmark after regression

**Comment by inspirit** [comment]
so it should be:
1. train classify only.
2. tune with classify + ROI
3. tune with classify + ROI + Landmarks
is it correct?

**Comment by CongWeilin** [comment]
yes

**Comment by CongWeilin** [comment]
finetune make it eaiser to see the result

**Comment by inspirit** [comment]
i wonder what is the average amount of iterations for each tune step?

**Comment by CongWeilin** [comment]
It depend on the trainning data... you can try yourself by watching the loss.

**Comment by inspirit** [comment]
ok, thank you
